eventmesh-common/src/main/java/org/apache/eventmesh/common/loadbalance/WeightRandomLoadBalanceSelector.java

```diff
@@ -74,4 +74,8 @@ public T select() {
     public LoadBalanceType getType() {
         return LoadBalanceType.WEIGHT_RANDOM;
     }
+
+    boolean isSameWeightGroup() {
+        return sameWeightGroup;
+    }
```

Comment:
How about using a `@Getter` lombok annotation?

Well, this method doesn't take up a lot of space and can be left unchanged in this PR if you don't feel it's necessary.